
<?xml version="1.0"?>
<launch>
  <!-- <node pkg="nodelet" type="nodelet" name="nodelet_manager" args="manager" /> -->

  <node pkg="nodelet" type="nodelet" name="nodelet1"
        args="load depth_image_proc/point_cloud_xyz nodelet_manager">
    <remap from="camera_info" to="/task1/masked_depth/camera_info"/>
    <remap from="image_rect" to="/task1/masked_depth/image"/>
    <remap from="points" to="/obstacle_cloud"/>
  </node>
  <node pkg="project" type="task1_depth_mask.py" name="task1_depth_mask" output="screen" />
</launch>